isis/templates/README.md

```diff
@@ -0,0 +1,29 @@
+# ISIS Templates
+
+The files in this directory are meant to be used as a starting point
+for developing new ISIS program files or guidelines for how existing
+files should be modified.
+
+
+## Generic Boilerplate for Most Files
+
+If a file has significant content (original material that authors
+could claim copyrights on) please ensure that the following text
+is placed in a comment near the top of the file, or in some other
+appropriate location, depending on file type.
+
+    This is free and unencumbered software released into the public domain.
+
+    The authors of ISIS do not claim copyright on the contents of this file.
+    For more details about the LICENSE terms and the AUTHORS, you will
+    find files of those names at the top level of this repository.
+
+Also include this as a separate comment underneath the above,
```

Comment:
Do you mean separate line?

Reply:
Whoops, sorry, I could have sworn I wrote something here, I even referenced it in the next comment down.

In Jay's C++ example, he had the SPDX line as a separate comment, that's how I implemented it in default.cpp, and that's what 'separate comment' was referring to.  Maybe we don't care, and we just have it all in one comment block, with the SPDX line separate in the example, and folks will just 'do the right thing' when they implement it, helped by specific examples?  I have no strong feelings on this, and am happy to take direction.

Reply:
I just made the change to remove that weird distinction, examples are better, anyway.